### ISSUE TITLE
FreeBSD support?

### ISSUE THREAD

**Issue by magiruuvelvet** [penpot/penpot#302]
This software seems to hard depend on docker and Linux. Is there any way to get this running on FreeBSD bare metal or other BSDs?

**Comment by niwinz** [comment]
Hello @magiruuvelvet 

Right now the application does not have hard dependency on docker. Docker is only used for development environment and is an option of deployment (our testing servers does not uses docker...)

I think the application can run perfectly on freebsd because the main project dependencies are JDK>=11, PostgreSQL, Redis and many system utilities (imagemagick, netpbm, etc...) I haven't tested it on freebsd but i think it should run on it perfectly.

**Comment by AtjonTV** [comment]
Do you guys plan to provide a bare-metal installation guide?

**Comment by niwinz** [comment]
Yes, of course. Internally we already have provision and deployment scripts using ansible for bare-metal or for plain vps. But currently we have all that mixed with private stuff. I think that in a near future we can put that scripts in the public repository. In any case the ansible scripts are based on debian stable and not freebsd, but freebsd variant will be accepted as PR.

**Comment by rajhlinux** [comment]
Using PenPot with chromium on FreeBSD 13.1

For some reason when zooming in, the computer instantly crashes and reboots.

Using AMD Radeon RX 580.

There is no docker on FreeBSD.

Would like to use PenPot on FreeBSD as offline.

**Comment by rajhlinux** [comment]
> Hello @magiruuvelvet
> 
> Right now the application does not have hard dependency on docker. Docker is only used for development environment and is an option of deployment (our testing servers does not uses docker...)
> 
> I think the application can run perfectly on freebsd because the main project dependencies are JDK>=11, PostgreSQL, Redis and many system utilities (imagemagick, netpbm, etc...) I haven't tested it on freebsd but i think it should run on it perfectly.

Can you make a tutorial on how to run it on FreeBSD?

Thanks.

**Comment by niwinz** [comment]
Dockerfiles are pretty descriptive on what should you need to run penpot. So anyone who want to have penpot outside docker, they just need to execute the same steps outside docker.

The step before, the penpot bundle for each module can be generated with `./manage.sh build-bundle` on the root of the repository.

**Comment by niwinz** [comment]
> Using PenPot with chromium on FreeBSD 13.1
> 
> For some reason when zooming in, the computer instantly crashes and reboots.
> 
> Using AMD Radeon RX 580.
> 
> There is no docker on FreeBSD.
> 
> Would like to use PenPot on FreeBSD as offline.

Looks like a chromium problem with the graphics subsystem of the operating system (freebsd in this case). ufortunatelly we can't do nothing to improve this, looks like it completelly unrelated to penpot. Is this still hapens when you run firefox on freebsd?

**Comment by yamila-moreno** [comment]
Hi there!
I'm closing this issue, because it's related to an old version; current latest (2.12.1) should not have this error. If you find it still happen with the newest version, feel free to reopen the issue.
Cheers!
